### ISSUE TITLE
git add issue

### ISSUE THREAD

**Issue by jianzhangbjz** [operator-framework/operator-lifecycle-manager#797]
@ecordell @njhale I get the below issue when running `git status`. I guess something wrong in the `git add` when commit.
```console
mac:operator-framework jianzhang$ git clone git@github.com:operator-framework/operator-lifecycle-manager.git
mac:operator-lifecycle-manager jianzhang$ git branch
* master
mac:operator-lifecycle-manager jianzhang$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   vendor/github.com/Sirupsen/logrus/.gitignore
	modified:   vendor/github.com/Sirupsen/logrus/CHANGELOG.md
	modified:   vendor/github.com/Sirupsen/logrus/README.md
	modified:   vendor/github.com/Sirupsen/logrus/entry.go
	modified:   vendor/github.com/Sirupsen/logrus/exported.go
	modified:   vendor/github.com/Sirupsen/logrus/formatter.go
	modified:   vendor/github.com/Sirupsen/logrus/go.mod
	modified:   vendor/github.com/Sirupsen/logrus/go.sum
	modified:   vendor/github.com/Sirupsen/logrus/json_formatter.go
	modified:   vendor/github.com/Sirupsen/logrus/logger.go
	modified:   vendor/github.com/Sirupsen/logrus/logrus.go
	modified:   vendor/github.com/Sirupsen/logrus/text_formatter.go
	modified:   vendor/github.com/Sirupsen/logrus/writer.go

no changes added to commit (use "git add" and/or "git commit -a")
```
And, `git add`/`git checkout --` didn't work.

**Comment by njhale** [comment]
This looks like you had a pre-existing vendor directory with changes. You could always run `git checkout -- .` to wipe out any tracked changes or even `git reset --hard HEAD` to wipe out all tracked/untracked changes.

**Comment by njhale** [comment]
Resolved, see #798
